api/main.py

```diff
@@ -1443,5 +1496,17 @@ async def carbondb_get_company_project_details(cptype: str, uuid: str):
 
     return ORJSONResponse({'success': True, 'data': data})
 
+# @app.get('/v1/authentication/new')
+# async def get_authentication_token(name: str = None):
+#     if name is not None and name.strip() == '':
+#         name = None
+#     return ORJSONResponse({'success': True, 'data': User.get_new(name)})
```

Comment:
This will fail if the DB insert fails but still report `'success': True`

Reply:
that code is not ready and just for future refernce. Added your comment thoug. ty